packages/components/src/components/forms/PageDetailHeader.tsx

```diff
@@ -55,6 +56,7 @@ export class PageDetailHeader extends React.Component<Props, any> {
                                 iconDir={iconDir}
                                 iconSrc={iconSrc ? iconSrc : ''}
                                 className="detail__header-icon"
+                                alt={iconAltText ? iconAltText : ''}
```

Comment:
Is this meant to be undefined if there is no iconAltText?  If the empty string is passed in we won't use iconSrc-icon as the alt text.

Reply:
the '' is falsy so the subsequent evaluation in SVGIcon should be correct.

http://es5.github.io/#x9.2

Reply:
Ah, OK. (Javascript is so fun.) Could be simplified to not use the ternary, but no big deal.